kvbc/src/Replica.cpp

```diff
@@ -600,7 +600,7 @@ Replica::Replica(ICommunication *comm,
   m_dbSet.dataDBClient->setAggregator(aggregator_);
   m_dbSet.metadataDBClient->setAggregator(aggregator_);
   auto stKeyManipulator = std::shared_ptr<storage::ISTKeyManipulator>{storageFactory->newSTKeyManipulator()};
-  m_stateTransfer = bftEngine::bcst::create(stConfig, this, m_metadataDBClient, stKeyManipulator, aggregator_);
+  m_stateTransfer.reset(bftEngine::bcst::create(stConfig, this, m_metadataDBClient, stKeyManipulator, aggregator_));
```

Comment:
So it looks like ST was nver called since we never called it's ctor, right?

Reply:
it is called later with "*m_stateTransfer", so the syntax persists.